src/components/organisms/news/news.scss

```
@@ -0,0 +1,38 @@
+@use "../../variables" as *;
+
+.last-assignment-news {
+  width: fit-content;
+  article {
+    width: 460px;
+    background-color: $white;
+    border: 1px solid $gray_gainsboro;
+    font-weight: 400;
+    font-family: $fontfamily-gothic;
+    .date {
+      font-size: $fontsize-xs;
+      color: $gray_nobel;
+      margin-top: 18px;
+      margin-left: 20px;
+    }
+    .article-title {
+      font-size: $fontsize-m;
+      margin-left: 20px;
+      margin-bottom: 18px;
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      transition: color 0.2s ease;
+      cursor: pointer;
+      &:hover {
+        color: $gray_nobel;
+      }
+    }
+  }
+  .news-and-events-more {
+    display: block;
+    margin-left: auto;
+    margin-right: 10px;
+    padding-top: 32px;
+    width: 24%;
```

クリックできる範囲にまで気をつかえていていい感じですね。
ここは fit-content にすると内容物ぴったりの幅になり、やりたいこととより近くなると思います。
（％だと、画面幅が変わった時に問題が出る可能性があります。）

```suggestion
    width: fit-content;
```